app/src/ingest_policy_pdfs.py

```diff
@@ -28,12 +42,12 @@ def _ingest_policy_pdfs(
     embedding_model = app_config.sentence_transformer
 
     logger.info(f"Processing pdfs {pdf_file_dir} using {embedding_model} with {doc_attribs}")
```

Comment:
Minor refactors here: rename file -> file_path, and fin -> file. Since we're not setting any LAParams, we can let pdfminer use its defaults rather than pass it ourselves.